app/services/bulk_upload/lettings/year2022/row_parser.rb

```diff
@@ -382,6 +385,28 @@ def property_ref
 
 private
 
+  def validate_created_by_exists
+    return if field_112.blank?
+
+    unless created_by
+      block_log_creation!
+      errors.add(:field_112, "User with the specified email could not be found")
```

Comment:
Do we want to block log creation if the user can't be found? My impression was that we would just import them silently and assign attribution to the author?

Reply:
just removed that now